proto/p4/config/v1/p4info.proto

```diff
@@ -56,6 +66,7 @@ message PkgInfo {
   // brief and detailed descriptions
   Documentation doc = 3;
   // Miscellaneous metadata, free-form; a way to extend PkgInfo
+  // raw annotations - deprecated, use annotation_map
   repeated string annotations = 4;
```

Comment:
I suggest adding the `[deprecated=true]` option to all `repeated string annotations` fields.

See https://developers.google.com/protocol-buffers/docs/proto3#options

Reply:
Great idea, done.